### PR TITLE
Update binary opcodes based on #51

### PR DESCRIPTION
**Pull request by ngzhian**

### CODE REVIEW

**Comment by ngzhian** [comment]
@tlively fyi for toolchain work
@dtig for V8
@yurydelendik  @lars-t-hansen for SpiderMonkey

**Comment by lars-t-hansen** [comment]
LGTM.

File: proposals/relaxed-simd/Overview.md [hunk 1]
```diff
@@ -211,34 +211,45 @@ implementation.
...
+| `i8x16.laneselect`                | 0x109          | 0xb2             |
+| `i16x8.laneselect`                | 0x10a          | 0xb3             |
+| `i32x4.laneselect`                | 0x10b          | 0xd2             |
+| `i64x2.laneselect`                | 0x10c          | 0xd3             |
+| `f32x4.min`                       | 0x10d          | 0xb4             |
+| `f32x4.max`                       | 0x10e          | 0xe2             |
+| `f64x2.min`                       | 0x10f          | 0xd4             |
+| `f64x2.max`                       | 0x110          | 0xee             |
+| `i16x8.q15mulr_s`                 | 0x111          | unimplemented    |
+| Reserved for dot product          | 0x112          | unimplemented    |
```
Comment (Maratyszcza) [inline proposals/relaxed-simd/Overview.md:241]: We have 4 relaxed integer dot product instructions + 1 upcoming BFloat16 dot product

Comment (ngzhian) [reply proposals/relaxed-simd/Overview.md]: Updated, thanks!

**Pull request merged**